int32 width
int32 height
float32 resolution
geometry_msgs/Point origin
int8[] costmap